packages/builder/src/components/backend/DatasourceNavigator/TableIntegrationMenu/IntegrationConfigForm.svelte

```diff
@@ -10,6 +10,7 @@
   import KeyValueBuilder from "components/integration/KeyValueBuilder.svelte"
   import { capitalise } from "helpers"
   import { IntegrationTypes } from "constants/backend"
+  import "@spectrum-css/accordion"
```

Comment:
Inserting these directly into the client/builder is generally a bad idea - if this were a component within BBUI it would fit better with the rest of our spectrum component design.

Reply:
Import removed!